### PR TITLE
add the ip host function for multiple static dns entries.

### PR DESCRIPTION
**Pull request by mmailand**

### CODE REVIEW

**Comment by jerearista** [comment]
Jenkins, test this please.

Martin, I will look at this in detail when I get back in the office next
week.

File: lib/puppet/provider/eos_iphost/default.rb [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+  # Mix in the api as instance methods
+  include PuppetX::Eos::EapiProviderMixin
+
+  # Mix in the api as class methods
+  extend PuppetX::Eos::EapiProviderMixin
+
+  def self.instances
+    hosts = node.api('iphosts').getall
+    #hosts = node.api('iphosts').get
```
Comment (jerearista) [inline lib/puppet/provider/eos_iphost/default.rb]: Please remove commented line

File: spec/unit/puppet/provider/eos_iphost/default_spec.rb [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+      it 'resource providers are absent prior to calling .prefetch' do
+        resources.values.each do |rsrc|
+          expect(rsrc.provider.ipaddress).to eq(:absent)
+        end
+      end
+
+      it 'sets the provider instance of the managed resource Host' do
+        subject
+        expect(resources['Host'].provider.name).to eq(@name)
+        # expect(resources['Host'].provider.ipaddress).to eq(@ipaddress)
```
Comment (jerearista) [inline spec/unit/puppet/provider/eos_iphost/default_spec.rb]: Why is this commented out?

**Comment by mmailand** [comment]
Hi Jere,
I changed the PR as requested.
Please review it.

**Pull request merged**